pkg/ccl/backupccl/restore_job.go

```diff
@@ -524,12 +524,13 @@ func WriteDescriptors(
 			return err
 		}
 
-		for _, table := range tables {
-			if err := table.TableDesc().Validate(ctx, txn, keys.SystemSQLCodec); err != nil {
-				return errors.Wrapf(err,
-					"validate table %d", errors.Safe(table.GetID()))
-			}
-		}
+		// TODO(mgartner): Another reason to support sqlbase.TableDescriptorInterface...
```

Comment:
Looks like `validator.ValidateTable` will have to have a `sqlbase.TableDescriptorInterface` argument instead of`*sqlbase.MutableTableDescriptor`. This will increase the complexity of the change because the interface doesn't yet have all the getters and setters that are necessary to perform validation.